docs/tutorial/part04.rst

```diff
@@ -2,5 +2,495 @@
 How to query the Neo4j database with Neomodel.
 ==============================================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Intro.
+======
+
+I assume that you know how to create a Django project and how to init and setup a Django app inside

```

Comment:
Avoid using sentences in first person use 'Assuming that' instead of 'I assume that' or if this was explained before you can use something like 'Now that you know...' or 'As explained in the previous section now you can create a Django project and init...' it will depend on the context.